src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -0,0 +1,671 @@
+        'url': url,
+        'device_a': device_a,
+        'device_b': device_b,
+        'pal_record': pal_record
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_PCR_1_default_config)
+  def test_WINNF_FT_S_PCR_1(self, config_filename):
+    # Load the Config file
+    config = loadConfig(config_filename)
+    url = config['url']
+    device_a = config['device_a']
+    pal_record = config['pal_record']
+    device_b = config['device_b']
+
+    # Step 1: Inject PAL record
+    self._sas_admin.InjectPalDatabaseRecord(pal_record)
+    # Step 2: register device_a in SAS UUT
+    self._sas_admin.InjectFccId({'fccId': device_a['fccId']})
+    self._sas_admin.InjectUserId({'userId': device_a['userId']})
+    request = {'registrationRequest': [device_a]}
+    response = self._sas.Registration(request)['registrationResponse'][0]
+    # Check registration response of device_a in SAS UUT
+    self.assertEqual(response['response']['responseCode'], 0)
+    del request, response
+    # Step 2: register device_b in SAS UUT
+    self._sas_admin.InjectFccId({'fccId': device_b['fccId']})
+    self._sas_admin.InjectUserId({'userId': device_b['userId']})
+    request = {'registrationRequest': [device_b]}
+    response = self._sas.Registration(request)['registrationResponse'][0]
+    # Check registration response of device_b in SAS UUT
+    self.assertEqual(response['response']['responseCode'], 0)
+    del request, response
+    # Step 3: Triggers SAS UUT to create a PPA boundary
+    cbsdIds = [device_a['cbsdSerialNumber'],device_b['cbsdSerialNumber']]
+    palIds = [pal_record['palId']]
+    response = self._sas_admin.TriggerPpaCreation(cbsdIds, palIds)
+    self.assertEqual(response['response']['responseCode'], 0)
+    self.assertNotEqual(response['ppaId'], None)
+    # Step 4: Triggers the SAS UUT to generate the Full Activity Dump
+    self._sas_admin.TriggerCreateActivityDump()
+    # Step 5: SAS Test Harness uses the SAS-SAS protocol to retrieve the PPA zone
+
+    full_activity_dump = _RequestGet('https://%s' %
+                (url + '/dump'))
+    self.polygon_dump = _RequestGet('https://%s' %
+                (full_activity_dump['url']))
+    # Step 6: Using PPA Creation Reference Model to calculate the maximum PPA
+    devices_list = [device_a, device_b]
+    pal_record_list = [pal_record]
+    pal_low_frequency = pal_record['channelAssign']['primaryAssignmen']['lowFrequency']
+    pal_high_frequency = pal_record['channelAssign']['primaryAssignmen']['highFrequency']
+    polygon_result = ppa_creation_model_byList(devices_list, pal_record_list, pal_record['userId'], pal_low_frequency, pal_high_frequency)
+    polygon = polygon_result['features'][0]['geometry']['coordinates'][0]
+    polygon = Polygon([tuple(i) for i in polygon])
+    polygon_dump = Polygon([tuple(i) for i in self.polygon_dump])
+    polygon_overlap = polygon.intersection(polygon_dump)
+    self.assertGreaterEqual(polygon_overlap.area, polygon.area * 0.9)
```

Comment:
This does not match the latest test spec.

Reply:
I think the test spec is to calculate the overlap of two polygons and make sure the intersection is equalorgreater 90%?

Reply:
Updated   the correct calculation based on latest test spec